audio/cog/commands/hybrid_commands.py

```diff
@@ -529,6 +531,105 @@ async def command_volume(self, context: PyLavContext, volume: Range[int, 0, 1000
+
+        player = context.lavalink.get_player(context.guild)
+        if not player:
+            await context.send(
+                embed=await context.lavalink.construct_embed(description=_("No player detected.")),
+                ephemeral=True,
+            )
+            return
+
+        if not player.current:
+            await context.send(
+                embed=await context.lavalink.construct_embed(description=_("Nothing playing.")),
+                ephemeral=True,
+            )
+            return
+
+        if not player.current.is_seekable:
+            if player.current.stream:
+                await context.send(
+                    embed=await context.lavalink.construct_embed(
+                        title=_("Unable to seek track"),
+                        description=_("Can't seek on a stream.")
+                    ),
+                    ephemeral=True,
+                )
+            else:
+                await context.send(
+                    embed=await context.lavalink.construct_embed(description=_("Unable to seek track.")),
+                    ephemeral=True,
+                )
+            return
+
+        if player.paused:
+            await context.send(
+                embed=await context.lavalink.construct_embed(description=_("Cannot seek when the player is paused.")),
+                ephemeral=True,
+            )
+            return
+
+        try:
+            seek = int(seek)
+            seek_ms = player.position + seek * 1000
+
+            if seek <= 0:
+                await context.send(
+                    embed=await context.lavalink.construct_embed(
+                        description=_("Moved {seconds}s to 00:00:00.").format(seconds=seek),
+                    ),
+                    ephemeral=True,
+                )
+            else:
+                await context.send(
+                    embed=await context.lavalink.construct_embed(
+                        description=_("Moved {seconds}s to {time}.").format(
+                            seconds=seek, time=format_time(seek),
+                        ),
+                    ),
+                    ephemeral=True,
+                )
+        except ValueError: #taken from https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/redbot/cogs/audio/core/utilities/miscellaneous.py#L197
+            match = _RE_TIME_CONVERTER.match(seek)
+            if match is not None:
+                hr = int(match.group(1)) if match.group(1) else 0
+                mn = int(match.group(2)) if match.group(2) else 0
+                sec = int(match.group(3)) if match.group(3) else 0
+                pos = sec + (mn * 60) + (hr * 3600)
+                seek_ms = pos * 1000
+            else:
+                seek_ms = 0
+
+            await context.send(
+                embed=await context.lavalink.construct_embed(
+                    description=_("Moved to {time}.").format(time=format_time(seek)),
```

Comment:
please make use of the messageable kwarg of the messageable and pass the context to it (this ensures we are using the bot colour config, on all your construct_embed calls